src/events/processCollectedSignatures/index.js

```diff
@@ -73,12 +71,12 @@ function processCollectedSignaturesBuilder(config) {
             gasEstimate = await estimateGas({
               foreignBridge,
               validatorContract,
+              signatures,
```

Comment:
Why do you pass `signatures` to the function if you already pass `v`, `r` and `s` components of these signatures?

Reply:
When I tried it in the repl, I couldn't make `eth.accounts.recover` work with `v`, `r` and `s`. But I will give it another try.

Reply:
Done.